### ISSUE TITLE
deploying on other evm chains

### ISSUE THREAD

**Issue by justinglibert** [ourzora/auction-house#4]
Hi there,
We'd like to use this protocol on xDai.
Do you want a canonical deployment from your team whose address would be on the readme or should we just go ahead and deploy it?
I am happy to PR the xDai address of the AuctionHouse contract  + verify the code on Blockscout (xDai's block explorer).

**Comment by tbtstl** [comment]
@justinglibert Do you know if the [Zora Core Protocol](github.com/ourzora/core) is already deployed on xDai? If it is, feel free to go ahead and deploy the AuctionHouse and let us know the addresses. We can add it to our list of addresses on docs.zora.co as well

**Comment by justinglibert** [comment]
I do not think it is deployed yet, but if it is admin-less I could deploy it now. I suppose the ethers factories are in a NPM package somehow?

**Comment by tbtstl** [comment]
Yes, it's easy enough to deploy. You'll just have to do the following:

1. Clone the [core repository](https://github.com/ourzora/core) locally
2. Create a file named `.env.local` with the following keys:
```
PRIVATE_KEY=
RPC_ENDPOINT=
```
3. Create a file called `100.json` in the `addresses/` directory with an empty JSON object (`{}`) as the initial content
4. Run `yarn && yarn deploy --chainId 100`
5. Create a PR into the core repository with your deployed addresses

**Comment by tbtstl** [comment]
@justinglibert Feel free to reach out on our discord if you run into issues, https://zora.community
